requirements.txt

```diff
@@ -0,0 +1,5 @@
+cortx-hare
+cortx-motr
+cortx-py-utils
+cortx-provisioner
```

Comment:
cortx-provisioner - Do we need provisioner ? I dont think we call provisioner. In fact provisioner calls us, so we can remove this as well.

Reply:
Not needed, removed.